resources/tx_curr_cql.txt

```diff
@@ -0,0 +1,25 @@
+library TX_CURR version '0.0.1'
+  
+using FHIR version '4.0.1'
+  
+include FHIRHelpers version '4.0.1' called FHIRHelpers
+  
+codesystem "CIEL": 'https://openconceptlab.org/orgs/CIEL/sources/CIEL'
+  
+code "Anti Retroviral Therapy": '160119' from "CIEL" display 'Currently taking ARV'
+ 
+code "Coded Answer": '1065' from "CIEL" display 'Yes'
+ 
+parameter "Measurement Period" Interval<DateTime>
+  
+context Patient
+  
+define "Initial population":
+    exists "Anti Retroviral Therapy Adults And Children"
+  
+define "Anti Retroviral Therapy Adults And Children":
+    ( [Observation: "Anti Retroviral Therapy"] ) o
+        where o.value.coding[1]= "Coded Answer"
+        or o.value.coding[2]= "Coded Answer"
+        where o.status in {'final', 'amended', 'corrected'}
```

Comment:
@gitcliff this is wrong.
this is not the cql you encoded. you cant have the `where` clause twice. it would fail